pinot-controller/src/test/resources/memory_estimation/schema-with-metadata.json

```diff
@@ -4,90 +4,90 @@
     {
       "dataType": "INT",
       "name": "colInt",
-      "cardinality": 100
+      "cardinality": 10
     },
     {
       "dataType": "INT",
       "name": "colIntMV",
       "singleValueField": false,
-      "cardinality": 150,
+      "cardinality": 15,
```

Comment:
Ok, so looks like this is the reason why the unit test results changed? Why do we need to alter the cardinality etc in this characteristics file ?

Reply:
Refer to the above comment.